voidcrew/modules/cloning/code/cloning_boards.dm

```diff
@@ -0,0 +1,46 @@
+//Computer
+
+/obj/item/circuitboard/computer/prototype_cloning
+	name = "Prototype Cloning"
+	greyscale_colors = CIRCUIT_COLOR_MEDICAL
+	build_path = /obj/machinery/computer/prototype_cloning
+
+/obj/item/circuitboard/computer/cloning
+	name = "Cloning"
+	greyscale_colors = CIRCUIT_COLOR_MEDICAL
+	build_path = /obj/machinery/computer/cloning
+
+//Machinery
+
+/obj/item/circuitboard/machine/clonepod
+	name = "Clone Pod"
+	greyscale_colors = CIRCUIT_COLOR_MEDICAL
+	build_path = /obj/machinery/clonepod
+	req_components = list(
+		/obj/item/stack/cable_coil = 2,
+		/obj/item/stock_parts/scanning_module = 2,
+		/obj/item/stock_parts/manipulator = 2,
+		/obj/item/stack/sheet/glass = 1)
+
+/obj/item/circuitboard/machine/clonepod/experimental
+	name = "Experimental Clone Pod"
+	greyscale_colors = CIRCUIT_COLOR_MEDICAL
+	build_path = /obj/machinery/clonepod/experimental
+
+//Machine Designs
+
+/datum/design/board/clonecontrol
```

Comment:
Just updated your branch to master, please move it to the rest of the designs as shown in https://github.com/Willardstation/tg-voidcrew/pull/79

Reply:
will do